corehq/blobs/tests/test_export.py

```diff
@@ -78,5 +85,121 @@ def test_migrate_multimedia(self):
         with NamedTemporaryFile() as out:
             exporter = EXPORTERS['multimedia'](self.domain_name)
             exporter.migrate(out.name, force=True)
-            with ZipFile(out.name, 'r') as zip:
-                self.assertEqual(expected, set(zip.namelist()))
+            with tarfile.open(out.name, 'r:gz') as tgzfile:
+                self.assertEqual(expected, set(tgzfile.getnames()))
+
+
+@skip('Uses over 33GB of drive space, and takes a long time')
```

Comment:
Why 33GB? Would a smaller size work?

Reply:
I chose it because it's 1GB more than the amount of RAM in my laptop. (If I reduce it to the amount of available RAM, that might work, or it might just push other stuff into swap.)

We could use `psutil` instead of hard-coding the size, so that it works on any machine:

      import psutil
      size = psutil.virtual_memory().total + 1

But the real issue is how long it takes -- well over half an hour! I think we could speed that up considerably by using 1MB blocks instead of generating 1 byte at a time, but I still think it would be much longer than we would want in Travis.

Let me know if you think I should pursue this further.

Reply:
The changes to use dynamic memory size look good. I also think it's fine to skip on Travis.